vscode-eslint/compile-eslint-server.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# @see https://github.com/microsoft/vscode-eslint
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="${SCRIPT_DIR}"
+SRC_DIR="${REPO_DIR}/checkout"
+SRC_SERVER_DIR="${SRC_DIR}/server"
+DIST_DIR="${REPO_DIR}/out"
+
+
+# ------------------------- #
+# download the source codes #
+# ------------------------- #
+
+pushd "${REPO_DIR}" || exit
+
+rm -rf \
+    "${SRC_DIR}" "${DIST_DIR}" \
+    "package.json" "package-lock.json"
+
+echo 'Enter commit SHA or tag (for example v2.1.0) to build'
+read -p 'SHA or tag: ' ref
+
+git clone https://github.com/microsoft/vscode-eslint "${SRC_DIR}"
```

Comment:
```suggestion
curl -L "https://github.com/microsoft/vscode-eslint/archive/${ref}.tar.gz" | tar -xzv
mv vscode-eslint-* "${SRC_DIR}"
```

Reply:
That one I need to be convinced about. :)
It's not like a checkout is big - it's something about 7MB. And it's more flexible since you can pass SHA or tag  and SHA doesn't have full lenght.

Reply:
actually partial SHA or tag works fine with that link

Reply:
Well, I'm convinced now :)

Reply:
The thing I've noticed now is that if you provide shorter SHA the download file will still have full one. So that creates a problem for moving .

Reply:
> The thing I've noticed now is that if you provide shorter SHA the download file will still have full one. So that creates a problem for moving .

probably can be 

```
mv "vscode-eslint-${ref}"* "${SRC_DIR}"
```

Reply:
Nice, thanks. I will do it the aggressive way